src/esp/bindings/PhysicsObjectBindings.cpp

```diff
@@ -315,6 +316,131 @@ void declareRigidObjectWrapper(py::module& m,
+                                     const std::string& objType,
+                                     const std::string& classStrPrefix) {
+  // ==== ManagedArticulatedObject ====
+  py::class_<ManagedArticulatedObject,
+             AbstractManagedPhysicsObject<ArticulatedObject>,
+             std::shared_ptr<ManagedArticulatedObject>>(
+      m, "ManagedArticulatedObject")
+      .def("create_joint_motor", &ManagedArticulatedObject::createJointMotor,
+           ("Create a joint motor for the specified DOF on this " + objType +
+            " using the provided JointMotorSettings")
+               .c_str(),
+           "DOF"_a, "settings"_a)
+      .def(
+          "remove_joint_motor", &ManagedArticulatedObject::removeJointMotor,
+          ("Remove the joint motor specified by the given motor_id from this " +
+           objType + ".")
+              .c_str(),
+          "motor_id"_a)
+      .def("get_joint_motor_settings",
+           &ManagedArticulatedObject::getJointMotorSettings,
+           ("Get the JointMotorSettings for the motor with the given "
+            "motor_id in this " +
+            objType + ".")
+               .c_str(),
+           "motor_id"_a)
+      .def("update_joint_motor", &ManagedArticulatedObject::updateJointMotor,
+           ("Update the JointMotorSettings for the motor specified by the "
+            "provided motor_id on this " +
+            objType + ".")
+               .c_str(),
+           "motor_id"_a, "settings"_a)
+      .def("get_link_scene_node", &ManagedArticulatedObject::getLinkSceneNode,
+           ("Get the scene node for this " + objType +
+            "'s articulated link specified by the passed "
+            "link_id. Use link_id==-1 to get the base link.")
+               .c_str(),
+           "link_id"_a)
+      .def("get_link_visual_nodes",
+           &ManagedArticulatedObject::getLinkVisualSceneNodes,
+           ("Get a list of the visual scene nodes from this " + objType +
+            "'s articulated link specified by the passed "
+            "link_id. Use link_id==-1 to get the base link.")
+               .c_str(),
+           "link_id"_a)
+      .def("get_link", &ManagedArticulatedObject::getLink,
+           ("Get this " + objType +
+            "'s articulated link specified by the passed "
+            "link_id. Use link_id==-1 to get the base link.")
+               .c_str(),
+           "link_id"_a)
+      .def(
+          "get_link_ids", &ManagedArticulatedObject::getLinkIds,
+          ("Get a list of this " + objType + "'s individual link ids.").c_str())
+      .def_property_readonly(
+          "num_links", &ManagedArticulatedObject::getNumLinks,
+          ("Get the number of links this " + objType + " holds.").c_str())
+      .def_property(
+          "forces", &ManagedArticulatedObject::getForces,
+          &ManagedArticulatedObject::setForces,
+          ("Get or set the forces acting upon this " + objType + ".").c_str())
```

Comment:
May not want a property for forces. Setting will call add torque to a total rather than set it. I see a couple of options. 
1. We add a new function which internally clears the torque before adding it.

2. We make this property read-only and also include an "addTorque" function.